signingscript/test/test_sign.py

```diff
@@ -284,6 +285,132 @@ def session_context():
     open_mock.assert_called()
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize('to,expected', ((
+    None, 'from',
+), (
+    'to', 'to'
+)))
+async def test_sign_hash_with_autograph(context, mocker, to, expected):
```

Comment:
I have an http error test case stashed, but couldn't figure out why the mocked response returned as None instead of being raised from the mocked retry async.